src/djangosaml2_spid/tests.py

```diff
@@ -129,10 +129,6 @@ def test_get_config(self):
         saml_config = get_config()
         self.assertIsNone(saml_config.entityid)
 
-        request = self.factory.get("")
```

Comment:
@brunato is there some particular need to have this kind of check? It looks to me quite pedant, please tell me how you put it there, probably is useful for something.

as you can see I also changed this behaviour to our default config_loader, 
tell me more when possibile, thx